include/lug/System/Utils.hpp

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+
+#if defined(LUG_COMPILER_MSVC)
+    #define LUG_SYSTEM_FUNCTION_NAME __FUNCSIG__
+#else
+    #define LUG_SYSTEM_FUNCTION_NAME __PRETTY_FUNCTION__
+#endif
+
+
+#if defined(__clang__) || defined(__GNUC__)
```

Comment:
I think that LUG_LIKELY and LUG_UNLIKELY must stay in Debug.hpp. When to use them outside the debug?